.github/workflows/test-fork-prs.yml

```diff
@@ -193,6 +193,50 @@ jobs:
         with:
           base-ref: ${{ needs.setup.outputs.base_sha }}
           head-ref: ${{ needs.setup.outputs.commit_id }}
+          allow-licenses:
```

Comment:
Can we have a reusable action for dependency review, and list all these licenses in one place?

Reply:
Talked with @wlee221 long time ago and we create a ticket to refactor this part into a reusable miscellaneous workflow.

Reply:
Well, actually I found this could be placed into a config. Let me try when I have some time 😄